### PR TITLE
Edit edit option to add order process modal.

### PR DESCRIPTION
**Pull request by Hyperkid123**
jira: https://projects.engineering.redhat.com/browse/SSP-1711

### Changes
- added edit variant to the add order process modal

Note: The order process and portfolio forms have exactly the same user flow and functionality. I will create a follow-up PR later which will merge these two components into a single component.

### CODE REVIEW

File: src/smart-components/order-process/add-order-process-modal.js [hunk 1]
```diff
@@ -8,41 +9,77 @@ import { ORDER_PROCESSES_ROUTE } from '../../constants/routes';
...
     </Modal>
   );
 };
 
+AddOrderProcess.propTypes = {
+  edit: PropTypes.bool
+};
+
+AddOrderProcess.defaultProps = {
+  edit: false
```
Comment (rvsia) [inline src/smart-components/order-process/add-order-process-modal.js:82]: Do you really need this default prop?

**Review by rvsia — COMMENTED** [review]
Just one code nitpick.

Otherwise, I found two issues:

**1**

image

I think the edit should be the first option.

**2**

When you enter the edit/add form, you lost the hash part of URL.

File: src/smart-components/order-process/add-order-process-modal.js (hunk 1 above)
Comment (Hyperkid123) [reply src/smart-components/order-process/add-order-process-modal.js:82]: not really. It's just more explicit.

**Comment by Hyperkid123** [comment]
@rvsia the order was changed and hash is now persistent. I would keep the default value.

**Review by rvsia — APPROVED** [review]
Tested the changes in UI, works great! :t-rex: Thank you!

**Pull request merged**